project/profiles/capacity4more/modules/c4m/user/c4m_user_notifications/c4m_user_notifications.module

```diff
@@ -123,8 +123,9 @@ function c4m_user_notifications_og_membership_update($og_membership) {
     if ($moderated == C4M_OG_MODERATED_GROUP) {
       // Check membership entity was an invitation or membership request.
       // If not invitation, at og_membership entity, og_membership_request
-      // field is an array with one item which is an empty array.
-      if (empty($og_membership->og_membership_request[0])) {
+      // field is either an empty array, or an array with one item which is
```

Comment:
please fix the docs so they'll actually explain about the scenario instead of the syntax.